src/main/java/teammates/logic/core/StudentsLogic.java

```diff
@@ -398,35 +398,96 @@ private void verifyIsWithinSizeLimitPerEnrollment(List<StudentAttributes> studen
         }
     }
 
-    public void validateSections(List<StudentAttributes> studentList, String courseId) throws EntityDoesNotExistException, EnrollException {
+    /**
+     * Validates both sections and teams for any limit violations
+     * @param studentList
+     * @param courseId
+     * @throws EntityDoesNotExistException
+     * @throws EnrollException
+     */
+    public void validateSectionsAndTeams(List<StudentAttributes> studentList, String courseId) throws EntityDoesNotExistException, EnrollException {
 
-        List<StudentAttributes> mergedList = new ArrayList<StudentAttributes>();
-        List<StudentAttributes> studentsInCourse = getStudentsForCourse(courseId);
+        List<StudentAttributes> mergedList = getMergedList(studentList, courseId);
+
+        if(mergedList.size() < 2){ // no conflicts
```

Comment:
Ensure that there are spaces surrounding the if statement i.e. `if (mergedList.size() < 2) {`